motion/core_ext/object/duplicable.rb

```diff
@@ -78,6 +78,6 @@ def duplicable?
 
 class BigDecimal
   def duplicable?
-    true
+    false
```

Comment:
I'm pretty sure RubyMotion targets parity with Ruby 1.9-ish, so this seems OK to me.  I'll open a ticket to point out that `dup` isn't implemented.